articles/portal/ptl-ref-release-notes.md

```diff
@@ -19,6 +19,16 @@ toc_mdlink: ptl-ref-release-notes.md
 
 This article provides a list of completed Portal releases.
 
+## Release 1.141.1
+
+Date: 24th June 2020
+
+SSO Workflow improvement
+
+Details:
+
+- Improved page flow when session timeout occurs
```

Comment:
Can we use the standard heading (e.g. "Feature Update" or "Bug fix"). Thanks, Michael

Reply:
Sorry Michael. I failed to recognise the established format. I hope it is correct now.